### DISCUSSION TITLE
Windows Remote client

### DISCUSSION THREAD

**Discussion by ghost** [containers/podman#10874] [Q&A]
Is the windows remote client no longer being built? I just tried to download it from the install docs / page link... and it said it cannot be found.

**Comment by Luap99** [comment]
@mheon PTAL, there are no binaries for https://github.com/containers/podman/releases/tag/v3.2.2

**Reply by mheon** [reply]
On it

**Reply by mheon** [reply]
Forgot to answer, but this is done
